internal/command/project_grant_member.go

```diff
@@ -107,19 +184,15 @@ func (c *Commands) removeProjectGrantMember(ctx context.Context, projectAgg *eve
 	}
 }
 
-func (c *Commands) projectGrantMemberWriteModelByID(ctx context.Context, projectID, userID, grantID string) (member *ProjectGrantMemberWriteModel, err error) {
+func (c *Commands) projectGrantMemberWriteModelByID(ctx context.Context, projectID, userID, grantID, resourceOwner string) (member *ProjectGrantMemberWriteModel, err error) {
 	ctx, span := tracing.NewSpan(ctx)
 	defer func() { span.EndWithError(err) }()
```

Comment:
Can't this be `defer span.EndWithError(err)` ?

Reply:
Didn't touch this, but theoretically yes.